### PR TITLE
Wito/feat/analytics opa

### PR DESCRIPTION
**Pull request by WitoDelnat**
This PR tracks OPA validation events.

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/utils/telemetry.ts [hunk 1]
```diff
@@ -19,6 +24,35 @@ export const trackError = (error: any) => {
...
+  CLUSTER_COMPARE: {numberOfResourcesBeingCompared: number} | {fail: string};
+  FOLLOW_LINK: {type: string};
+  QUICK_SEARCH: undefined;
+  UPDATE_APPLICATION: undefined;
+  APPLY_FILE: undefined;
+  APPLY_HELM_CHART: undefined;
+  RUN_PREVIEW_CONFIGURATION: undefined;
+  VALIDATION_PANE_OPENED: {id: ValidationIntegrationId};
+  OPA_ENABLED: {all: boolean};
+  OPA_DISABLED: {all: boolean};
```
Comment (WitoDelnat) [inline src/utils/telemetry.ts:53]: I wonder if our analytics platform works well enough to work with these kind of parametrised events or whether we need to split it up in OPA_ENABLED and OPA_ALL_ENABLED.

Comment (devcatalin) [reply src/utils/telemetry.ts:53]: it works well, we can send any object as a payload as long as it doesn't contain functions

File: src/utils/telemetry.ts [hunk 2]
```diff
@@ -2,13 +2,18 @@ import {ipcRenderer} from 'electron';
 
 import Nucleus from 'nucleus-nodejs';
 
+import {ValidationIntegrationId} from '@components/organisms/ValidationDrawer/integrations';
```
Comment (devcatalin) [inline src/utils/telemetry.ts:5]: here we might have the problem with importing `.tsx` to main process?

Comment (WitoDelnat) [reply src/utils/telemetry.ts:5]: Hmm you are correct. Damnit, I just fixed that in my other branch. Really annoying limitation. What do you think if I add jsx to TS compiler and we only use `import type`? I think it's unlikely anyone will use React components in Node process.

Comment (WitoDelnat) [reply src/utils/telemetry.ts:5]: Actually this is just a `.ts` file so it works properly. We can reconsider this if we actually run into it again but for this PR it is fine.

**Review by devcatalin — APPROVED** [review]
LGTM

**Pull request merged**